src/Microsoft.ServiceFabric.Diagnostics/Microsoft.ServiceFabric.Diagnostics.csproj

```diff
@@ -4,7 +4,9 @@
 
   <PropertyGroup>
       <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-      <AssemblyName>Microsoft.ServiceFabric.Diagnostics</AssemblyName>
+      <RootNamespace>Microsoft.ServiceFabric.Diagnostics</RootNamespace>
+      <RootNamespace>$(AssemblyName)</RootNamespace>
+      <AssemblyTitle>$(AssemblyName)</AssemblyTitle>
```

Comment:
This doesn't look right. Is the AssemblyName now inferred from the project name? 

The RootNamespace is assigned twice and only the second assignment is in effect.

I don't think we need the AssemblyTitle if it's the same as AssemblyName. It's just a metadata property/attribute that plays only informative role for humans. 

This looks like a copy from other projects. Let's use this opportunity to understand these properties and simplify the projects by leaving only the meaningful ones.

Reply:
Either way is fine. Your call. 😄